schemas/context/admeasuressearch-adcloud.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/admeasuressearch-adcloud",
```

Comment:
Classes are intended to have universal meaning across the platform. Is this class for an adcloud purpose only? If so, should it be moved under the adcloud namespace instead? 
What does this class do (or not do) that the existing experienceevent class does not provide?

Reply:
Agreed, this should be in ad cloud namespace.  Maybe there was some mechanical reason this couldn't be done that we need to address?

Reply:
Moved Ad Cloud Specific class under ad-cloud namespace.  The underlying data-set doesn't fall under experience event category (data is not time-series events rather aggregated data of number of clicks within a day). Hence we had to model as a ad-cloud search specific class.